docs/podman.1.md

```diff
@@ -36,6 +36,10 @@ Note: CGroup manager is not supported in rootless mode when using CGroups Versio
 
 Path to where the cpu performance results should be written
 
+**--events-logger**=**type**
```

Comment:
Nit: the convention everywhere else seems to be to use italics (one asterisk) for the arg value.